contracts/ethregistrar/ETHRegistrarController.sol

```diff
@@ -1,178 +1,278 @@
-        uint expires = base.renew(uint256(label), duration);
+        PriceOracle.Price memory price = rentPrice(name, duration);
+        require(
+            msg.value >= price.base,
+            "ETHController: Not enough Ether provided for renewal"
+        );
 
-        if(msg.value > cost) {
-            payable(msg.sender).transfer(msg.value - cost);
-        }
+        uint256 expires = base.renew(uint256(label), duration);
 
-        emit NameRenewed(name, label, cost, expires);
-    }
+        if (msg.value > price.base) {
+            payable(msg.sender).transfer(msg.value - price.base);
+        }
 
-    function setPriceOracle(PriceOracle _prices) public onlyOwner {
-        prices = _prices;
-        emit NewPriceOracle(address(prices));
+        emit NameRenewed(name, label, msg.value, expires);
     }
 
-    function setCommitmentAges(uint _minCommitmentAge, uint _maxCommitmentAge) public onlyOwner {
-        minCommitmentAge = _minCommitmentAge;
-        maxCommitmentAge = _maxCommitmentAge;
+    function withdraw() public {
+        payable(owner()).transfer(address(this).balance);
     }
 
-    function withdraw() public onlyOwner {
-        payable(msg.sender).transfer(address(this).balance);        
+    function supportsInterface(bytes4 interfaceID)
+        external
+        pure
+        returns (bool)
+    {
+        return
+            interfaceID == type(IERC165).interfaceId ||
+            interfaceID == type(IETHRegistrarController).interfaceId;
     }
 
-    function supportsInterface(bytes4 interfaceID) external pure returns (bool) {
-        return interfaceID == INTERFACE_META_ID ||
-               interfaceID == COMMITMENT_CONTROLLER_ID ||
-               interfaceID == COMMITMENT_WITH_CONFIG_CONTROLLER_ID;
-    }
+    /* Internal functions */
 
-    function _consumeCommitment(string memory name, uint duration, bytes32 commitment) internal returns (uint256) {
-        // Require a valid commitment
-        require(commitments[commitment] + minCommitmentAge <= block.timestamp);
+    function _consumeCommitment(
+        string memory name,
+        uint256 duration,
+        bytes32 commitment
+    ) internal {
+        // Require a valid commitment (is old enough and is committed)
+        require(
+            commitments[commitment] + minCommitmentAge <= block.timestamp,
+            "ETHRegistrarController: Commitment is not valid"
+        );
 
         // If the commitment is too old, or the name is registered, stop
-        require(commitments[commitment] + maxCommitmentAge > block.timestamp);
-        require(available(name));
+        require(
+            commitments[commitment] + maxCommitmentAge > block.timestamp,
+            "ETHRegistrarController: Commitment has expired"
+        );
+        require(available(name), "ETHRegistrarController: Name is unavailable");
 
-        delete(commitments[commitment]);
+        delete (commitments[commitment]);
```

Comment:
Why the space?

Reply:
My prettier auto added that on a fresh install of VSCode + solidity plugin. I'm not sure how to get rid of it